ocfweb/urls.py

```diff
@@ -28,6 +29,7 @@
 
     url(r'^$', home, name='home'),
     url(r'^robots\.txt$', robots_dot_txt, name='robots.txt'),
+    url(r'^sign\.json$', sign_text, name='sign.json'),
```

Comment:
I think this should be under a `/api` namespace or something, just to make it clear it isn't for general use, and to make a place where other API-only URLs can be put.

Same with the location of the file above (in `ocfweb/api/sign/sign_text.py` or something)